builders/cordova-serve/index.ts

```diff
@@ -52,6 +53,14 @@ class CordovaDevServerBuilder extends DevServerBuilder {
     super(context);
   }
 
+  run(builderConfig: BuilderConfiguration<DevServerBuilderOptions>): Observable<BuildEvent> {
+    if (this.cordovaBuildOptions.consolelogs && this.cordovaBuildOptions.consolelogsPort) {
+      createConsoleLogServer(builderConfig.options.host, this.cordovaBuildOptions.consolelogsPort)
```

Comment:
If you wanted to get a little fancier, you can use `fromPromise`: http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#static-method-fromPromise

Reply:
I didn't want to actually slow the build down waiting for the server to spin up by chaining it with the call to `super`, is that what you meant? Something like `from(createConsoleLogServer(...)).pipe(_ => super.run(builderConfig))`? My observable foo isn't great. 

I'm going to leave it as is for now so let me know if you had something different in mind.